test/engineEvaluate.test.ts

```diff
@@ -0,0 +1,132 @@
+import { testables } from '../src/engine/evaluate';
+import {
+  makeEngineCondition,
+  makeQuery,
+  makePageView,
+} from './helpers/engineConditions';
+import { QueryFilterComparisonType } from '../types';
+
+const { filterPageViews } = testables;
+
+const multipleOrthogonalQueriesCondition = makeEngineCondition(
```

Comment:
Whilst this is the correct mathematical definition - I am not sure this is the best way to describe the code in this context.

Reply:
`multipleNonOverlapingQueryConditions` maybe?